### PR TITLE
updates for GCC high/Dod.md

### PR DESCRIPTION
**Pull request by srividhc**

### CODE REVIEW

**Comment by LolaJacobsen** [comment]
DO NOT merge this PR until Tuesday morning, October 15, 2019. Lola will merge it at that time.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ See these articles to plan and set up the Phone System in your business:
 
 - [Set up Calling Plans](set-up-calling-plans.md): This article has step-by-step instructions to get set up with your phone numbers or to transfer your existing phone numbers to Skype for Business.
     
-- [Microsoft telephony solutions](https://docs.microsoft.com/en-us/SkypeForBusiness/hybrid/msft-telephony-solutions): This  article lays out all of your deployment options. It also has information about planning an entirely cloud-only solution. 
+- [Microsoft telephony solutions](https://docs.microsoft.com/SkypeForBusiness/hybrid/msft-telephony-solutions): This  article lays out all of your deployment options. It also has information about planning an entirely cloud-only solution.
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
- [Microsoft telephony solutions](https://docs.microsoft.com/SkypeForBusiness/hybrid/msft-telephony-solutions): This  article lays out all of your deployment options. It also has information about planning an entire cloud-only solution. 
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 2]
```diff
@@ -95,7 +95,17 @@ See these articles to plan and set up the Phone System in your business:
...
-   
+
+## Availability in GCC High and DoD clouds
+<a name="bkmk_setup"> </a>
+
+The following capabilites are not yet available in GCC High and DoD Clouds. 
+- [Call Settings for secondary ringer, voicemail and enhanced delegation](https://support.office.com/article/Manage-your-call-settings-in-Teams-456cb611-3477-496f-b31a-6ab752a7595f)
+- [Transfer to voicemail mid call](https://support.office.com/article/Transfer-a-call-in-Teams-b7f40f14-e083-46b9-b739-68038c8f73a0)
+- Call Phone number from search bar
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
- Call phone number from search bar
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 3]
```diff
@@ -95,7 +95,17 @@ See these articles to plan and set up the Phone System in your business:
...
+- Music on hold
+- AAD Reverse number lookup
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
- AAD reverse number lookup
```

**Pull request merged**